draft-ietf-shmoo-cancel-meeting.md

```diff
@@ -28,7 +28,7 @@ informative:
 
 The IETF holds three in-person meetings per year to discuss and understand
 issues. However, various emergencies can make a planned in-person meeting
-infeasible. This document provides criteria for making this judgment.
+infeasible. This document provides criteria for assessing meating feasibility.
```

Comment:
```suggestion
infeasible. This document provides criteria for assessing meeting feasibility.
```

Reply:
Oops, how embarassing!
Though perhaps in my defense I can claim that MITbeef holds meatings...